app/views/static_pages/cookies.html.erb

```diff
@@ -66,6 +66,11 @@
         </tr>
       </thead>
       <tbody>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell">accept_cookie_policy</td>
+          <td class="govuk-table__cell">This lets us know that you have accepted our cookie policy</td>
+          <td class="govuk-table__cell">After 1 year</td>
+        </tr>
```

Comment:
This is the table for Google Analytics. This should be in its own section, I think.